frontend/packages/dev-console/src/components/pipelines/pipelineForm-validation-utils.ts

```diff
@@ -0,0 +1,17 @@
+import * as yup from 'yup';
```

Comment:
pls move this to utils/pipelines/

Reply:
Not sure if it should be moved there since this will be only used for form components. Although, I have created a separate folder for pipeline-form so that the validations and form can stay at one place.